foodsaving/invitations/serializers.py

```diff
@@ -45,3 +47,20 @@ class InvitationAcceptSerializer(serializers.Serializer):
     def update(self, invitation, validated_data):
         invitation.accept(self.context['request'].user)
         return invitation
+
+
+class InvitationResendEmailSerializer(serializers.Serializer):
+    from datetime import timedelta
+
+    created_at = serializers.DateTimeField()
+
+    def validate(self, attrs):
```

Comment:
Why you pass the date from frontend to backend? The backend already knows about it (`invitation.created_at`) and the frontend is generally untrustworthy. It would be better to remove this.

Reply:
I thought this was the best approach since I would have to pass the data manually to the serializer, but now that the serializer is not needed, that won't be a problem anymore.